src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -173,26 +173,48 @@ trait ClusterComponent extends LeoComponent {
         }
     }
 
-    def getByGoogleId(googleId: UUID): DBIO[Option[Cluster]] = {
-      clusterQueryWithInstancesAndErrorsAndLabels.filter { _._1.googleId === googleId }.result map { recs =>
+    // TODO: Check if this should be removed and its usages replaced by getById
```

Comment:
I would vote removing it. In general we should only have `getByX` methods where X is unique and has an index. Usages of `getByGoogleId` should probably be replaced with `getById`.